azure-pipelines.yml

```diff
@@ -111,11 +111,17 @@ stages:
           inputs:
             command: custom
             arguments: 'locals all -clear'
-        - script: eng\common\cibuild.cmd -configuration $(_BuildConfig) -prepareMachine $(_BuildArgs)
+        - script: eng\common\cibuild.cmd
+            -configuration $(_BuildConfig)
+            -prepareMachine
+            $(_BuildArgs)
+            /bl:artifacts/log/build.windows.binlog
           displayName: Build and Publish
         - script: eng\scripts\ci-flaky-tests.cmd -configuration $(_BuildConfig)
           displayName: Run Flaky Tests
           continueOnError: true
+        - script: dotnet msbuild eng/repo.targets /t:EnsureFunctionalTestLogsPreserved
```

Comment:
I suppose I can add the verification to macOS and Linux runs as well, but I don't think it adds much.

Reply:
Meh